### PR TITLE
Update release_guide.md

### PR DESCRIPTION
**Pull request by reutshal**
## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added instructions regarding the master build from the SDK pre-release

### CODE REVIEW

File: docs/release_guide.md [hunk 1]
```diff
@@ -2,12 +2,17 @@ In order to release a new version of `demisto-sdk` to the public follow these st
...
-5) Look at the CircleCI build:
-  a) Check that the unit-tests container passed sucessfully.
-  b) Check that the Test Playbooks have started running successfully, and that no erorrs occurred when setting the Demisto server instance.
-  c) Cancel the build as we do not want to run all the nightly test playbooks.
+3) Update the version of the SDK in Demisto's Content repository by updating the demisto-sdk version in the [**dev-requirements-py3.txt**](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt) file. Use the release branch first - replace the `demisto-sdk==version` line with this line: `git+https://github.com/demisto/demisto-sdk.git@release-branch-name.`
+4) As the regular Content build runs, initiate the following builds:  
+  a) Trigger the nightly Content build from the Content repository by running:  
+  `./Utils/trigger_content_nightly_build.sh <branch_name> <circle_token>`.  
+  b) Trigger a build that imitates the content master build from the SDK repository by running:   `./demisto_sdk/utils/trigger_against_content_master.sh <release_branch_name> <circle_token>`
```
Comment (ronykoz) [inline docs/release_guide.md:9]: why do we need them both?

Comment (reutshal) [reply docs/release_guide.md:9]: To my understanding the flows are different. 
Until now only nightly was running.

Comment (ronykoz) [reply docs/release_guide.md:9]: yeah but we are not changing anything in the tests, if we are already mimicking the commands in out build why should we do anything apart from it?

@yuvalbenshalom @Itay4 @yaakovi what do you think

Comment (reutshal) [reply docs/release_guide.md:9]: @ronykoz any new thoughts?

Comment (yaakovi) [reply docs/release_guide.md:9]: currently, triggering nightly has no special features besides running all tests (unit tests and test playbooks).
after content files will be valid in a way we can run validate with `--validate-all` flag, I think we should use it in nightly builds.
this change should be included under the new `validate.sh` script so triggering the special build in the SDK will cover it.

File: docs/release_guide.md [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
-13) Announce regarding the SDK release in the content-team slack channel.
+1) Make sure that sdk-nightly and sdk-master builds are both green.
+2) Make sure the **CHANGELOG.md** file is in order and is updated with all the changes in the current release.
+3) In demisto-sdk, create a new release branch, named after the release version in the format `X.X.X`, e.g. `1.0.0`
+4) Important! If there were more commits to the sdk after the nightly tests finished, you will need to perform the tests manually before the release:  
+  a. Trigger the nightly-sdk build from the Content repository by running:  
+     `./Utils/trigger_nightly_sdk_build.sh <branch_name> <circle_token>`.  
+  b. Trigger a build that imitates the content master build from the SDK repository by running:   
+     `./demisto_sdk/utils/trigger_against_content_master.sh <release_branch_name> <circle_token>`.  
+  Make sure both builds are green before relasing.
```
Comment (yaakovi) [inline docs/release_guide.md:11]:
```suggestion
  Make sure both builds are green before relasing.
```

File: docs/release_guide.md [hunk 3]
```diff
@@ -1,21 +1,27 @@
...
+4) Important! If there were more commits to the sdk after the nightly tests finished, you will need to perform the tests manually before the release:  
+  a. Trigger the nightly-sdk build from the Content repository by running:  
+     `./Utils/trigger_nightly_sdk_build.sh <branch_name> <circle_token>`.  
+  b. Trigger a build that imitates the content master build from the SDK repository by running:   
+     `./demisto_sdk/utils/trigger_against_content_master.sh <release_branch_name> <circle_token>`.  
+  Make sure both builds are green before releasing.
+5) Update the version of the SDK in Demisto's Content repository by updating the demisto-sdk version in the [**dev-requirements-py3.txt**](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt) file. Use the release branch first - replace the `demisto-sdk==version` line with this line: `git+https://github.com/demisto/demisto-sdk.git@release-branch-name.`
+6) If there was any change in the content creation steps (**create content artifacts** or **unify** commands) - we need to check that the new content is valid.
+To do so, you can do the followings:
```
Comment (yaakovi) [inline docs/release_guide.md:14]:
```suggestion
To do so, you can do the following:  
```

File: docs/release_guide.md [hunk 4]
```diff
@@ -1,21 +1,27 @@
...
+  Make sure both builds are green before releasing.
+5) Update the version of the SDK in Demisto's Content repository by updating the demisto-sdk version in the [**dev-requirements-py3.txt**](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt) file. Use the release branch first - replace the `demisto-sdk==version` line with this line: `git+https://github.com/demisto/demisto-sdk.git@release-branch-name.`
+6) If there was any change in the content creation steps (**create content artifacts** or **unify** commands) - we need to check that the new content is valid.
+To do so, you can do the followings:  
+  a. Compare the new_content.zip from nightly-sdk build and nightly-content build and see if there is any major difference between them.  
+  b. If needed, trigger the nightly Content build from the Content repository by running:    
+  `./Utils/trigger_content_nightly_build.sh <branch_name> <circle_token>` and make sure to wait until the build is finished.
+7) In demisto-sdk repository's main page click on **releases**.
+8) Click on **Draft a new release**.
+9) Update the **Tag version** and **Release title** - the form should be vX.X.X .
```
Comment (yaakovi) [inline docs/release_guide.md:20]:
```suggestion
9) Update the **Tag version** and **Release title** - the form should be `vX.X.X` .
```

File: docs/release_guide.md [hunk 5]
```diff
@@ -1,21 +1,27 @@
...
+  a. Trigger the nightly-sdk build from the Content repository by running:  
+     `./Utils/trigger_nightly_sdk_build.sh <branch_name> <circle_token>`.  
+  b. Trigger a build that imitates the content master build from the SDK repository by running:   
+     `./demisto_sdk/utils/trigger_against_content_master.sh <release_branch_name> <circle_token>`.  
+  Make sure both builds are green before releasing.
+5) Update the version of the SDK in Demisto's Content repository by updating the demisto-sdk version in the [**dev-requirements-py3.txt**](https://github.com/demisto/content/blob/master/dev-requirements-py3.txt) file. Use the release branch first - replace the `demisto-sdk==version` line with this line: `git+https://github.com/demisto/demisto-sdk.git@release-branch-name.`
+6) If there was any change in the content creation steps (**create content artifacts** or **unify** commands) - we need to check that the new content is valid.
+To do so, you can do the following:  
+  a. Compare the new_content.zip from nightly-sdk build and nightly-content build and see if there is any major difference between them.
```
Comment (ronykoz) [inline docs/release_guide.md:15]:
```suggestion
  a. Compare the content_new.zip from nightly-sdk build and nightly-content build and see if there is any major difference between them.  
```

**Pull request merged**